guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -19,6 +19,55 @@ The advantage of this is that each organization maintains separate subscriptions
 You can use future-dated subscriptions in a subscription manifest.
 When you add future-dated subscriptions to your manifest before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.
 
+.Benefits of the subscriptions service
+* Tracks selected Red Hat product usage and capacity at the fleet or account level in a unified inventory and provides a daily snapshot of that data in a digestible, filterable dashboard at cloud.redhat.com.
+* Tracks data over time for self-governance and analytics that can inform purchasing and renewal decisions, ongoing capacity planning, and mitigation for high-risk scenarios.
+* Helps procurement officers make data-driven choices with portfolio-centered reporting dashboards that show both inventory-occupying subscriptions and current subscription limits across the entire organization.
+* With its robust reporting capabilities, enables the transition to Simple Content Access (SCA) tooling that features broader, organizational-level subscription enforcement instead of system-level quantity enforcement.
+
+Simple Content Access (SCA) is a capability in Red Hat’s subscription tools which simplifies the behavior of the entitlement tooling, making it easier to consume the content provided by your Red Hat subscriptions without the complexity of configuring subscription tooling.
+
+The subscriptions service, hosted at console.redhat.com provides a means for customers to view their subscription usage and can show usage data from both connected and disconnected environments if configured to do so.
+
+.Configuring your system to use the subscription services
+
+Connected (Non-{Project}) users::
+
+As a connected, non-{Project} user, no further action needs to be taken to use the subscriptions service. 
+For more information, see https://console.redhat.com/subscriptions/usage/rhel to track your usage. 
+Registering a system through the guidance in https://docs.redhat.com/en/documentation/subscription_central/1-latest/html-single/getting_started_with_rhel_system_registration/index[Getting Started with RHEL System Registration] is sufficient to ensure the system is properly counted. 
+If necessary, set https://access.redhat.com/articles/system-purpose[System Purpose] attributes to denote SLA and USAGE to better power filtering.
+
+Connected {Project} Users::
+
+Connected {Project} Users should enable the `foreman_rh_cloud` plugin if it hasn’t been enabled already (https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_the_subscriptions_service/proc-installing-satellite-inventory-upload-plugin_assembly-setting-up-subscriptionwatch-ctxt). The foreman_rh_cloud plugin shares inventory with the Hybrid Cloud Console enabling the usage of the subscriptions service to track usage information.
```

Comment:
[RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#contractions) says to avoid contractions unless the context is informal (example: marketing).

Reply:
This may need an if-ndef for Satellite. foreman_rh_cloud is included with Satellite and enabled by default, so this sentence would only apply to upstream.